l1-contracts/deploy-scripts/upgrade/SystemContractsProcessing.s.sol

```diff
@@ -262,6 +262,12 @@ library SystemContractsProcessing {
             lang: Language.Solidity,
             isPrecompile: false
         });
+        systemContracts[32] = SystemContract({
+            addr: 0x0000000000000000000000000000000000010008,
+            codeName: "ProxyAdmin",
```

Comment:
no, we dont need to force deploy it anywhere, besides this may override the real 10008 `L2_INTEROP_ROOT_STORAGE_ADDRESS` contract. Please ONLY publish factory dependency (look at how it was done with bridged erc20 token, etc)